src/main/java/team4/footwithme/member/repository/MemberRepository.java

```diff
@@ -4,6 +4,9 @@
 import org.springframework.stereotype.Repository;
 import team4.footwithme.member.domain.Member;
 
+import java.util.Optional;
+
 @Repository
 public interface MemberRepository extends JpaRepository<Member, Long> {
+    Optional<Member> findByEmail(String email);
```

Comment:
```suggestion
    Member findByEmail(String email);
```

이렇게 작성하고 서비스단에서 예외처리하는걸 선호하는데 Optional을 사용해도 null check를 해야지 예외처리가 가능하다고 생각하는데 이부분은 어떻게 생각하시나요 ?

Reply:
저는 보통 Optional 받고 orElseThrow() 로 예외처리를 해주는데 이 때 null 값이 들어가면 예외를 던지고 아니면 값을 받아오는 걸로 알고 있어서 사용을 해왔는데 민혁님은 null 값을 if문으로 체크해주시는 걸까요??

Reply:
아하 코드 다시보니까 저랑 똑같이 쓰시네요 재경님은 Optional<Member>로 하고 실제로 받을 때는 Member로 받네요 이런식으로 받아도 문제가 없나보네요

저는 여기서 Optional<Member> 이렇게 받으면 서비스에서도 Optional<Member>로 받아야되는줄 알았어요 Optional을 잘 안써서.. 그래서 전 Optional<Member>로 받고 null 체크할바에는 Member로 작성하고 예외처리하는게 더 나아보였는데 제가 잘 몰랐던 부분 같아요
이 부분은 제가 Optional을 더 공부해야되겠네요.. 감사합니다